File: CPP/beginner/1010.cpp
#include <stdio.h>

int main() {
    int FirstCode = 0, SecondCode = 0, FirstUnit = 0, SecondUnit = 0;
    double FirstPrice = 0, SecondPrice = 0, EndPrice = 0;

    scanf("%d %d %lf", &FirstCode, &FirstUnit, &FirstPrice);

    scanf("%d %d %lf", &SecondCode, &SecondUnit, &SecondPrice);

    EndPrice = (FirstUnit * FirstPrice) + (SecondUnit * SecondPrice);
  
    printf("VALOR A PAGAR: R$ %.2lf\n", EndPrice);

    return 0;
}

File: CSharp/beginner/1010.cs
using System;

class Program
{
    static void Main(string[] args)
    {
        string[] firstProduct = Console.ReadLine().Split(' ');
        int firstCode = int.Parse(firstProduct[0]);
        int firstQuantity = int.Parse(firstProduct[1]);
        double firstPricel = double.Parse(firstProduct[2]);

        string[] secondProduct = Console.ReadLine().Split(' ');
        int secondCode = int.Parse(secondProduct[0]);
        int secondQuantity = int.Parse(secondProduct[1]);
        double secondPricel = double.Parse(secondProduct[2]);

        double valueTotal = (firstQuantity * firstPricel) + (secondQuantity * secondPricel);

        Console.WriteLine($"VALOR A PAGAR: R$ {valueTotal:F2}");
    }
}
